internal/cli/root.go

```diff
@@ -264,6 +264,46 @@ func (c *Client) NewCommand() *cobra.Command {
 	return c.rootCmd
 }
 
+func (c *Client) MetalListOptions(defaultIncludes, defaultExcludes []string) (incl, excl []string, sea, sBy, sDir string) {
+	var inc, exc []string
+	var search, sortBy, sortDir string
```

Comment:
Each of these should have a separate function.  `ListOptions` is a packngo concept and doesn't make sense for `metal-go`.  By having separate functions, you can do things like `request.Include(c.Servicer.Includes()).Excludes(c.Servicer.Excludes()).Search(c.Servicer.Search())`.

Reply:
updated